app/server/lib/DocApi.ts

```diff
@@ -926,16 +927,16 @@ export class DocWorkerApi {
 
         const docId = activeDoc.docName;
         const webhookId = req.params.webhookId;
-        const {fields, url} = await getWebhookSettings(activeDoc, req, webhookId, req.body);
+        const {fields, url, authorization} = await getWebhookSettings(activeDoc, req, webhookId, req.body);
         if (fields.enabled === false) {
           await activeDoc.triggers.clearSingleWebhookQueue(webhookId);
         }
 
         const triggerRowId = activeDoc.triggers.getWebhookTriggerRecord(webhookId).id;
 
-        // update url in homedb
+        // update url and authorization header in homedb
         if (url) {
-          await this._dbManager.updateWebhookUrl(webhookId, docId, url);
+          await this._dbManager.updateWebhookUrlAndAuth(webhookId, docId, url, authorization || "");
```

Comment:
Should authorization be set to empty string? Or should it be set to `null` to express its absence?

I'd suggest to remain consistant with the rest of the code and allow it to be `null` or `undefined`